src/Views/Reference/NavalFlagTable.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { NavalFlags, CharacterImage } from 'puzzle-lib';
+import { Table } from 'react-bootstrap';
+import './NavalFlagTable.css';
+
+class NavalFlagTable extends React.Component {
+  public render() {
+    return (
+      <div className="NavalFlagsTable">
+        <div className="NavalFlagsTable-content">
+          <Table striped={true} responsive={true}>
+            <thead>
+              <tr>
+                <th>Letter</th>
+                <th>Flag</th>
+              </tr>
+            </thead>
+            <tbody>
+              {this.renderRows(NavalFlags.instance.entries)}
+            </tbody>
+          </Table>
+        </div>
+      </div>
+    );
+  }
+
+  private renderRows(data: CharacterImage[]) {
+    return data.map((value: CharacterImage) => {
+      const width = 100;
```

Comment:
Add a className to the img tag and set this in CSS.  See https://github.com/kfarnung/puzztool/commit/b2ac463a9e1af704c3c1172b2d1d0810290ebd45

Reply:
Done.